src/apps/uniswap-v2/common/uniswap-v2.pool.subgraph.types.ts

```diff
@@ -26,6 +26,18 @@ export const DEFAULT_LAST_BLOCK_SYNCED_ON_GRAPH_QUERY = gql`
   }
 `;
 
+export const FURA_LAST_BLOCK_SYNCED_ON_GRAPH_QUERY = gql`
+  {
+    indexingStatusForCurrentVersion(subgraphName: "quickswap") {
```

Comment:
`subgraphName` should be a parameter, extract it from the last part of the subgraph URL after the last `/`